codex/sales/states/downloading.nim

```diff
@@ -31,6 +31,11 @@ method onSlotFilled*(state: SaleDownloading, requestId: RequestId,
                      slotIndex: UInt256): ?State =
   return some State(SaleFilled())
 
+# override onError so that the slot is reprocess in the case of an unhandled
```

Comment:
```suggestion
# override onError so that the slot is reprocessed in the case of an unhandled
```

Reply:
Good catch, will remove the entire override